bundles/tools.mdsd.characteristics.model/model/api.genmodel

```diff
@@ -1,23 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <genmodel:GenModel xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel"
-    modelDirectory="/tools.mdsd.characteristics.model/src-gen" creationIcons="false"
-    editDirectory="/tools.mdsd.characteristics.model.edit/src-gen" editorDirectory="/tools.mdsd.characteristics.model.editor/src-gen"
-    modelPluginID="tools.mdsd.characteristics.model" modelName="Api" editPluginClass=""
-    editorPluginClass="" rootExtendsClass="org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container"
+    modelDirectory="/tools.mdsd.characteristics.model/src-gen" editDirectory="/tools.mdsd.characteristics.edit/src-gen"
+    editorDirectory="/tools.mdsd.characteristics.editor/src-gen" modelPluginID="tools.mdsd.characteristics.model"
+    modelName="Api" editPluginClass="tools.mdsd.characteristics.CharacteristicsEditPlugin"
+    editorPluginClass="tools.mdsd.characteristics.CharacteristicsEditorPlugin" rootExtendsClass="org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container"
```

Comment:
We got some problems opening the editors for the characteristics model due to NPE in CharacteristicsEditorPlugin. The editor plugin class is different for api.genmodel and characteristics.genmodel.

Reply:
should be fixed now. Proper fix follows through api restructuring.